src/parser/mod.rs

```diff
@@ -617,6 +617,9 @@ impl<'a> Parser<'a> {
                 }
                 // `COMMENT` is snowflake specific https://docs.snowflake.com/en/sql-reference/sql/comment
                 Keyword::COMMENT if self.dialect.supports_comment_on() => self.parse_comment(),
+                Keyword::PRINT if dialect_of!(self is MsSqlDialect) => {
```

Comment:
```suggestion
                Keyword::PRINT => {
```
we can parse unconditionally?

Reply:
Done 👍